tests/perf/actor_id_scale/test.js

```diff
@@ -0,0 +1,55 @@
+import http from "k6/http";
+import { check } from "k6";
+import exec from "k6/execution";
+
+const defaultMethod = "default"
+const actorType = __ENV.ACTOR_TYPE
```

Comment:
ALL_CAPS?

Reply:
it is common to have env vars as caps. ps: the __ENV is from the k6 framework